Checkstyle error:
```
> Task :streams:checkstyleMain
07:47:53 [ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk14-scala2.13/streams/src/main/java/org/apache/kafka/streams/processor/internals/ForwardingDisabledProcessorContext.java:43:33: Name 'explanation' must match pattern '(^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$)|(^log$)'. [ConstantName]
```

--- a/streams/src/main/java/org/apache/kafka/streams/processor/internals/ForwardingDisabledProcessorContext.java
+++ b/streams/src/main/java/org/apache/kafka/streams/processor/internals/ForwardingDisabledProcessorContext.java
@@ -40,7 +40,7 @@ import java.util.Objects;
 public final class ForwardingDisabledProcessorContext implements ProcessorContext {
     private final ProcessorContext delegate;
 
-    private static final String explanation = "ProcessorContext#forward() is not supported from this context, "
+    private static final String EXPLANATION = "ProcessorContext#forward() is not supported from this context, "
         + "as the framework must ensure the key is not changed (#forward allows changing the key on "
         + "messages which are sent). Try another function, which doesn't allow the key to be changed "
         + "(for example - #tranformValues).";
@@ -107,24 +107,24 @@ public final class ForwardingDisabledProcessorContext implements ProcessorContex
 
     @Override
     public <K, V> void forward(final K key, final V value) {
-        throw new StreamsException(explanation);
+        throw new StreamsException(EXPLANATION);
     }
 
     @Override
     public <K, V> void forward(final K key, final V value, final To to) {
-        throw new StreamsException(explanation);
+        throw new StreamsException(EXPLANATION);
     }
 
     @Override
     @Deprecated
     public <K, V> void forward(final K key, final V value, final int childIndex) {
-        throw new StreamsException(explanation);
+        throw new StreamsException(EXPLANATION);
     }
 
     @Override
     @Deprecated
     public <K, V> void forward(final K key, final V value, final String childName) {
-        throw new StreamsException(explanation);
+        throw new StreamsException(EXPLANATION);
     }
 
     @Override

